api/serializers/team.py

```diff
@@ -74,6 +91,20 @@ def create(self, validated_data):
         member = validated_data.get('member')
         team.users.remove(member)
 
-        send_remove_email.delay(team.id, member.id)
+        data = self.prepare_email(team.id, member.id)
+
+        send_email.delay(**data)
 
         return validated_data
+
+    def prepare_email(self, team_id, user_id):
```

Comment:
I would recommend create new module with name email and prepare/send email by this module. 
For example: 
1. Create class TeamEmail.
2. TeamEmail has: send_remove_notification, send_add_notification.
3. these functions do: prepare data for email and send email.
4. And here you can call only these function.